spec.bs

```diff
@@ -1586,6 +1593,24 @@ an [=interest group=] |ig|, a [=boolean=] |isComponentAuction| and a [=boolean=]
     [=InterestGroupReportingScriptRunnerGlobalScope/report url=] to |parsedUrl|.
 </div>
 
+<div algorithm>
+  The <dfn method for=Navigator>registerAdBeacon(|map|)</dfn> method steps are:
+
+  1. If [=this=]'s [=relevant global object=]'s [=InterestGroupReportingScriptRunnerGlobalScope/
+     reporting beacon map=] is not null, then [=exception/Throw=] a {{TypeError}}.
+
+  1. [=map/For each=] |url| of |map|'s [=map/values=]:
+
+     1. Let |parsedURL| be the result of running [=URL parser=] on |url|.
+
+     1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
+        * |parsedURL| is an error;
+        * |parsedURL|'s [=url/scheme=] is not "`https`".
+
+  1. Set [=this=]'s [=relevant global object=]'s [=InterestGroupReportingScriptRunnerGlobalScope/

```

Comment:
The relevant global object here is a Window, since Navigator (the interface that this method hangs off of) is in a window, not a FLEDGE "worket", so I think this does not make sense. Should this method not be `for=Navigator` but instead for the reporting script runner global worklet?